gs/spec/bfs/bucket/gs_spec.rb

```diff
@@ -6,37 +6,38 @@ def warn_if_cloud_sdk_credentials(*); end
   module_function :warn_if_cloud_sdk_credentials # rubocop:disable Style/AccessModifierDeclarations
 end
 
-sandbox = { project: 'bogus', bucket: 'bsm-bfs-unittest' }.freeze
```

Comment:
Now it will write to default project? IMO, that's a bit unsafe. Though S3 spec seems to be done this way, so I guess it's OK.

Reply:
project was ignored anyway, as there is no `bogus` project (in our ownership)